<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Vigenere</class>
 <widget class="QMainWindow" name="Vigenere">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="VigenereCipher">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Vigenere Cipher</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="TTCN">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Danh Minh Quá»‘c - 2280618774</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="PlainText">
      <property name="text">
       <string>Plain_text:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="plaintext_txt"/>
    </item>
    <item>
     <widget class="QLabel" name="Key">
      <property name="text">
       <string>Key:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="key_txt"/>
    </item>
    <item>
     <widget class="QLabel" name="CipherText">
      <property name="text">
       <string>Cipher_text:</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="ciphertext_txt"/>
    </item>
    <item>
     <widget class="QPushButton" name="decrypt_btn">
      <property name="text">
       <string>Decrypt</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="encrypt_btn">
      <property name="text">
       <string>Encrypt</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
